src/commands/ImportCommand.js

```diff
@@ -0,0 +1,88 @@
+import fs from "fs";
+import path from "path";
+import child from "child_process";
+import Command from "../Command";
+import progressBar from "../progressBar";
+import PromptUtilities from "../PromptUtilities";
+
+export default class ImportCommand extends Command {
+  initialize(callback) {
+    const inputPath = this.input[0];
+
+    if (!inputPath) {
+      callback(new Error("Missing argument: Path to external repository"));
+    }
+
+    const externalRepoPath = path.resolve(inputPath);
+
+    try {
+      const stats = fs.statSync(externalRepoPath);
+      if (!stats.isDirectory()) {
+        throw new Error(`Input path "${inputPath}" is not a directory`);
+      }
+      const packageJson = path.join(externalRepoPath, "package.json");
+      const packageName = require(packageJson).name;
```

Comment:
Do we want to make a specific error message here or no? (if the package path doesn't have a package.json)

```
Errored while running ImportCommand.initialize
Error: Cannot find module '/module/package.json'
```